hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/precompile/proxy/ViewExecutorTest.java

```diff
@@ -296,13 +309,17 @@ void computeGetNonFungibleTokenInfo() {
                 .when(() -> NonFungibleTokenInfoPrecompile.decodeGetNonFungibleTokenInfo(input))
                 .thenReturn(tokenInfoWrapper);
 
-        given(stateView.infoForToken(nonfungibletoken)).willReturn(Optional.of(tokenInfo));
+        given(stateView.getNetworkInfo()).willReturn(networkInfo);
+        given(networkInfo.ledgerId()).willReturn(ByteString.copyFromUtf8("0xff"));
```

Comment:
nit: add ledgerId ByteString to class static fields since it's done so many times

Reply:
DONE